cachito/web/migrations/versions/4d17dec0cfc3_adjust_unique_constraint.py

```diff
@@ -0,0 +1,29 @@
+"""Adjust unique constraint on Flag table
+Remove unique constraint from id and name but keep name unique
```

Comment:
Let's use a single line description, I think we can just keep the first one.

I would also suggest to rename the file `adjust_unique_constraint` -> `adjust_flag_unique_constraint`